tests/README.md

```diff
@@ -1,53 +1,63 @@
 
-The tests run using both the plugin elf and the ethereum elf, both compiled for
-Nano (S, S plus, X) and Stax devices, so 8 binaries are needed. The plugin binaries are expected to be stored in the
-`tests/lib_binaries/` directory, and named as follow:
 
-* `tests/lib_binaries/pluginBoilerplate_nanos.elf`
-* `tests/lib_binaries/pluginBoilerplate_nanos2.elf`
-* `tests/lib_binaries/pluginBoilerplate_nanox.elf`
-* `tests/lib_binaries/pluginBoilerplate_stax.elf`
+## Binaries
+
+In order to run the tests you need to provide the binaries for both the plugin and the Ethereum application.
 
-The ethereum binaries are expected to be stored in the
-`build/` directory, and named as follow:
+Compiling the plugin will produce the binary for the targeted device. They will be used as-is by the test framework.
 
-* `build/nanos/bin/app.elf`
-* `build/nanos2/bin/app.elf`
-* `build/nanox/bin/app.elf`
-* `build/stax/bin/app.elf`
+The binaries of the Ethereum application must be gathered and placed in the directory `tests/ethereum_build/`.
+Example of the correct file tree with Ethereum compiled for all targets.
+* `tests/ethereum_build/build/nanos/bin/app.elf`
+* `tests/ethereum_build/build/nanos2/bin/app.elf`
+* `tests/ethereum_build/build/nanox/bin/app.elf`
+* `tests/ethereum_build/build/stax/bin/app.elf`
 
-This [CI](https://github.com/LedgerHQ/app-plugin-boilerplate/blob/develop/.github/workflows/build_and_functional_tests.yml) can be used to generate the binaries artifacts.
+The first method is to go in the Ethereum project, compile the application, and dispatch the `build/` output directory.
+A second method is to re-use the Ethereum build used in the CI, and available as artifact. 
+For example in the [plugin-boilerplate CI](https://github.com/LedgerHQ/app-plugin-boilerplate/actions/workflows/build_and_functional_tests.yml).
 
 
 ## Launching the tests
 
-The plugin boilerplate app comes with functional tests implemented with Ledger's [Ragger](https://github.com/LedgerHQ/ragger) test framework.
+The plugin boilerplate app comes with functional tests 
+
 
 ### macOS / Windows
 
 To test your app on macOS or Windows, it is recommended to use [Ledger's VS Code extension](#with-vscode) to quickly setup a working test environment.
 
-You can use the following task (all accessible in the **extension sidebar menu**) :
+You can use the following task (accessible in the **extension sidebar menu**) :
 
 * Use `Run tests`.
 
 ### Linux (Ubuntu)
 
-On Linux, you can use [Ledger's VS Code extension](#with-vscode) to run the tests. If you prefer not to, open a terminal and follow the steps below.
+On Linux, you can use [Ledger's VS Code extension](#with-vscode) to run the tests.
+
+Alternatively, you can run the test by command lines in your terminal.
 
-Install the tests requirements :
+First, run this command once to install the tests requirements:
 
 ```shell
-pip install -r tests/requirements.txt 
+pip install -r tests/requirements.txt --extra-index-url
```

Comment:
Probably better to remove `--extra-index-url` and to add these lines in requirements.txt
```
--index-url https://test.pypi.org/simple/
ledger_app_clients.ethereum
```

Reply:
I thought we decided to keep the command line to be explicit ? And handle it in readme (for manual usage) and vscode plugin
cc @apaillier-ledger